services/app/src/components/modal/UserModal.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+import Modal from 'react-modal';
+
+const customStyles = {
+  content: {
+    top: '50%',
+    left: '50%',
+    right: 'auto',
+    bottom: 'auto',
+    marginRight: '-50%',
+    transform: 'translate(-50%, -50%)',
+  },
+};
+const UserModal = () => {
+  const [modalIsOpen, setIsOpen] = useState(false);
+
+  const closeModal = () => {
+    setIsOpen(false);
+  };
+
+  const submitInfo = (event) => {
+    event.preventDefault();
+    window.localStorage.setItem('user', event.target.username.value);
+    window.location.reload();
+  };
+
+  return (
+    <div>
+      <Modal
```

Comment:
Do we really need a Modal here? 

Looking at the code, it seems like the Modal is not really what we need here, as this modal is used on its own page, without any interactions by the user, that would open / close the modal. I would expect the modal to be used when we want to have some sort of an overlay over another content. We don't need a modal when the modal is the only component on the screen and its always open, a couple of styled `div`s would do the job just fine.

Looking at Figma designs for Landing Page, it seems like the login form does can be simply placed on its own screen as it isn't displayed over any other content in the background.

Let's get rid of this `react-modal` dependency for now, use plain HTML to implement login form and restructure the routes as I've described in the above comment.